### ISSUE TITLE
Ghosting after resize

### ISSUE THREAD

**Issue by yoav-lavi** [helix-editor/helix#8093]
### Summary

I seem to occasionally get "ghosting" effects after resizing the terminal / full screen. This normally manifests in text lines that are "stuck" and don't refresh or duplicated lines

### Reproduction Steps

 

I tried this:

1. `hx`
2. Rapidly resize the terminal (WezTerm)

I expected this to happen:

Text remain in the correct position

Instead, this happened:

Ghosting effects

<img width="667" alt="Screenshot 2023-08-28 at 13 04 49" src="https://github.com/helix-editor/helix/assets/14347895/7decc47a-0472-485c-a72f-6b01c3728f9f">

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>
Will add this later on if relevant

</details>

### Platform

macOS

### Terminal Emulator

wezterm 20230712-072601-f4abf8fd

### Helix Version

9f843e4

**Comment by yoav-lavi** [comment]
Note: I haven't currently been able to reproduce this outside of a `zellij` session so it may be related to running within `zellij`

**Comment by pascalkuthe** [comment]
yeah this seems like a zellij bug and not ahelix but, I can not reproduce this, you should report this to zellij
